src/App.tsx

```diff
@@ -15,36 +17,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  SORT_ALPHABET = 'SORT_ALPHABET ',
```

Comment:
The `SORT_ALPHABET` value in the `SortType` enum has an extra space at the end. This could cause issues when comparing `sortField` values. Remove the trailing space to fix this.